.github/workflows/main.yml

```diff
@@ -72,7 +72,20 @@ jobs:
 
   build:
```

Comment:
Now run build in parallel with linting and tests, so this doesn't make the pipeline slower